framework/include/parser/Syntax.h

```diff
@@ -68,7 +69,7 @@ class Syntax
                            int line = -1);
 
   /**
-   *  Registration a type with a block. For example, associate FunctionName with the Functions block
+   * Registration a type with a block. For example, associate FunctionName with the Functions block
```

Comment:
Want to fix the "Registration a type" -> "Register a type" typo on this line as well?